### ISSUE TITLE
Node v6.9.2 error but works on earlier versions

### ISSUE THREAD

**Issue by ghost** [cgross/grunt-dom-munger#42]
Updated to node v6.9.2 and now get the following error in a yo generator that includes a Gruntfile.js

```
Running "dom_munger:read" (dom_munger) task

Processing index.html
Warning: Path must be a string. Received [ 'index.html' ] Use --force to continue.
```

I've made no changes to the Gruntfile.js and this is the config of dom_munger read object:

```
            read: {
                options: {
                    read: [
                        {selector: '[build-concat]', attribute: 'src', writeto: 'jsRefs', isPath:true}
                    ]
                },
                src: 'index.html'
            }, 
```

Any suggestions?

**Comment by andrewblaney** [comment]
For info: Removing isPath:true fixed this issue

**Comment by jrgleason** [comment]
So is this the legitimate answer? Should we change `path.dirname(grunt.file.expand(f))` to handle an array as well as a string?

**Comment by hagabaka** [comment]
Pull requests #44 and #45 are both meant to fix this.

**Comment by niemyjski** [comment]
We've fixed this in 4.0 :)
